### DISCUSSION TITLE
unexpected error in interpolating bands in wannier berri

### DISCUSSION THREAD

**Discussion by subhadipniser333** [wannier-berri/wannier-berri#207] [Q&A]
Hi wannier-berri developers,
                                          I am trying to do the tutorial "interpolating bands, Berry curvatures, and integrating them" as directed by you for my input files for a material. I am facing a few problems. Firstly in 6 the step, i.e., in "# plot the bands and compare with QuantumEspresso," I have tried to plot the bands from vasp output. When I am trying to change the energy limit, i.e., Emin and Emax from -2 to 2, it is showing "UnboundLocalError: local variable '_line' referenced before assignment," but when I am trying with '-5' to '5' range it is showing proper band structure. Secondly, the wannier bands' energy windows are in negative how do I get the energy window of my choice? Also, the wannier band is not matching with vasp bands. In the berry curvature plot, the energy window for the bands is very small and negative. How to change this energy range?

Thanks,
Subhadip

**Comment by stepan-tsirkin** [comment]
It is hard to understand what is happening without seeing your figures.

It looks like when you set to a narrow window, there might be no bands in that range Note also the `Eshift` parameter.

**Comment by subhadipniser333** [comment]
Thanks. The issue is resolved.

**Reply by subhadipniser333** [reply]
Hi,
   I am getting the berry curvature projected band structure like this fat band from wannier-berri,
![Screenshot from 2022-10-16 12-22-11](https://user-images.githubusercontent.com/83715114/196022476-f547a433-c5a2-4b4b-b02f-3ccef6485061.png)
But can I get a plot like this one with a color bar by changing the source code anyway,
![Screenshot from 2022-10-16 12-19-50](https://user-images.githubusercontent.com/83715114/196022641-f30bf782-cde3-4bc3-981a-377b51489be3.png).
Thanks,
Subhadip
